modules/mediabramaBidAdapter.md

```diff
@@ -0,0 +1,33 @@
+# Overview
+
+```
+Module Name: MediaBrama Bidder Adapter
+Module Type: MediaBrama Bidder Adapter
+Maintainer: support@mediabrama.com
+```
+
+# Description
+
+Module that connects to mediabrama demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'div-prebid',
+            mediaTypes:{
+                banner: {
+                    sizes: [[300, 250]],
+                }
+            },
+            bids:[
+                {
+                    bidder: 'mediabrama',
+                    params: {
+                        placementId: '24428' //test, please replace after test 
```

Comment:
Could you work on returning a valid response from your endpoint for your bidders example params? This is helpful for us to ensure your adapter works as expected in auctions

Reply:
respond has been fixed.

Reply:
Thanks, seeing 200 but they are empty responses. We'll need to see a bid return in the format your adapter is expecting, this helps others confirm your adapter behaves as expected.

Reply:
<img width="323" alt="image" src="https://github.com/prebid/Prebid.js/assets/145676355/9e88a05f-319c-4ab0-9a3a-c668e992bffb">
We have fixed it.